src/main/java/athena/Main.java

```diff
@@ -0,0 +1,86 @@
+package athena;
+
+import athena.commands.Command;
+import athena.commands.ShutdownCommand;
+import athena.exceptions.InputException;
+import athena.parser.Parser;
+import athena.storage.Storage;
+import athena.tasks.TaskList;
+import athena.ui.Ui;
+
+import java.io.IOException;
+
+/**
+ * Provides the point of entry into the Athena program. Initializes the program and handles
+ * the main program logic.
+ */
+public class Main {
+    private static final String SAVE_DIRECTORY = "data";
+    private static final String SAVE_FILENAME = "athena.txt";
+
+    private TaskList taskList;
+    private final Storage storage;
+    private final Ui ui;
+    private boolean isActive;
+
+    /**
+     * Initializes the program by loading task data from disk if any and initializing all
+     * requisite objects and data structures.
+     */
+    public Main() {
+        storage = new Storage(SAVE_DIRECTORY, SAVE_FILENAME);
```

Comment:
I like how you declared the file path for memory as static constants, this makes it more convenient if there is a need to change!